app/experimenter/static/js/components/DesignForm.js

```diff
@@ -46,15 +52,25 @@ class DesignForm extends React.PureComponent {
       value: "",
       is_control: false,
     });
+    const emptyErrorsBranch = fromJS({
```

Comment:
Sorry I don't think the multiline comment worked right? What I meant was `emptyBranch` and `emptyErrorsBranch` are almost exactly the same, do they need to be declared separately?

Reply:
What I am realizing is that we can just add an empty object to errors, it doesn't need to have any keys.